week-3/A-array-find/exercise.js

```diff
@@ -1,13 +1,18 @@
-/* 
-  You are given an array of test scores.
-  Any result over 60 is a pass. 
-  Using .find() (and any other array methods necessary), find the lowest pass score
+/*
+  You are given an array of names.
+  Using .find(), we'd like to find the first name which starts with A and is longer than 7 letters.
 */
 
-var testScores = [90, 50, 100, 25, 81, 66, 80];
-var lowestPassScore = findLowestPassScore(testScores);
+// write your code here
 
-console.log(lowestPassScore);
+var names = ["Rakeshss", "Alexandra", "Annam", "Mikey", "Karim", "Ahmed"];
+var longNameThatStartsWithA = names.find(findLongNameThatStartsWithA);
+
+function findLongNameThatStartsWithA(names) {
+  return names.length >= 7 && names[0] === "A";
```

Comment:
This is really [nitpicking](https://dictionary.cambridge.org/dictionary/english/nitpicking), but the exercise asks you to find names that are longer than 7 letters, not equal to or longer than 7 letters.

Also what happens if you change the names array so that "Alexandra" is spelled like "alexandra"? How could we fix this?